server/src/main/java/io/druid/curator/discovery/DiscoveryModule.java

```diff
@@ -170,6 +187,20 @@ public void configure(Binder binder)
             .addBinding(CURATOR_KEY)
             .to(CuratorDruidNodeAnnouncer.class)
             .in(LazySingleton.class);
+
+    PolyBind.optionBinder(binder, Key.get(DruidLeaderSelector.class, Coordinator.class))
+            .addBinding(CURATOR_KEY)
+            .toProvider(new DruidLeaderSelectorProvider(
+                (zkPathsConfig) -> ZKPaths.makePath(zkPathsConfig.getCoordinatorPath(), "druid:coordinator"))
```

Comment:
why is this hard coded here?

Reply:
it is same as https://github.com/druid-io/druid/blob/master/server/src/main/java/io/druid/server/coordinator/DruidCoordinator.java#L525f  just inlined.

Reply:
(minor) since the path is explicitly for druid, the `druid:` is redundant. Can it retain the prior naming mechanism of `_COORDINATOR`?

Reply:
yeah, changed ... using `_COORDINATOR` and  `_OVERLORD` now